pkg/asset/installconfig/powervs/platform.go

```diff
@@ -0,0 +1,128 @@
+        //
+
+        longRegions := make([]string, 0, len(regions))
+	shortRegions := make([]string, 0, len(regions))
+	for id, location := range regions {
+		longRegions = append(longRegions, fmt.Sprintf("%s (%s)", id, location))
+		shortRegions = append(shortRegions, id)
+	}
+        
+	var regionTransform survey.Transformer = func(ans interface{}) interface{} {
+		switch v := ans.(type) {
+		case core.OptionAnswer:
+			return core.OptionAnswer{Value: strings.SplitN(v.Value, " ", 2)[0], Index: v.Index}
+		case string:
+			return strings.SplitN(v, " ", 2)[0]
+		}
+		return ""
+	}
+
+        // TODO(cklokman): We need to verify that us_south is the correct defaultRegion to use
+        //
+	defaultRegion := "us_south"
+	if !IsKnownRegion(defaultRegion) {
+		panic(fmt.Sprintf("installer bug: invalid default powervs region %q", defaultRegion))
+	}
+
+	ssn, err := GetSession()
+	if err != nil {
+		return nil, err
+	}
+
+	defaultRegionPointer := ssn.Config.Region
+	if defaultRegionPointer != nil && *defaultRegionPointer != "" {
+		if IsKnownRegion(*defaultRegionPointer) {
+			defaultRegion = *defaultRegionPointer
+		} else {
+			logrus.Warnf("Unrecognized powervs region %q, defaulting to %s", *defaultRegionPointer, defaultRegion)
+		}
+	}
+
+	sort.Strings(longRegions)
+	sort.Strings(shortRegions)
+
+	var region string
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Select{
+				Message: "Region",
+				Help:    "The powervs region to be used for installation.",
+				Default: fmt.Sprintf("%s (%s)", defaultRegion, regions[defaultRegion]),
+				Options: longRegions,
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
+				choice := regionTransform(ans).(core.OptionAnswer).Value
+				i := sort.SearchStrings(shortRegions, choice)
+				if i == len(shortRegions) || shortRegions[i] != choice {
+					return errors.Errorf("invalid region %q", choice)
+				}
+				return nil
+			}),
+			Transform: regionTransform,
+		},
+	}, &region)
+	if err != nil {
+		return nil, err
+	}
+
+        var zoneTransform survey.Transformer = func(ans interface{}) interface{} {
+                switch v := ans.(type) {
+                case core.OptionAnswer:
+                        return core.OptionAnswer{Value: strings.SplitN(v.Value, " ", 2)[0], Index: v.Index}
+                case string:
+                        return strings.SplitN(v, " ", 2)[0]
+                }
+                return ""
+        }
+        
+        var zone string
+        err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Select{
+				Message: "Zone",
+				Help:    "The powervs zone within the region to be used for installation.",
+				Default: fmt.Sprintf("%s", defaultZone),
+				Options: longZones,
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
+				choice := zoneTransform(ans).(core.OptionAnser).Value
+				i := sort.SearchStrings(shortZones, choice)
```

Comment:
so will the masters/workers not be spread across zones? is the concept of a zone in  powervs different from other clouds? i mean the obvious reason to have them across zones is for failover.

Reply:
i mean shouldn't this be a list of zones?

Reply:
@Prashanth684 -- I'm trying to find out for sure, but I don't think that by default we support multi-zone regions in Power VS. There may be additional tunneling required. For now, I think we can put this as a todo. Unless @bpradipt can answer definitively.

Reply:
by support i should have clarified communication between regions. 
also wanted to drop this here as well: https://github.com/IBM-Cloud/power-go-client#ibm-cloud-sdk-for-power-cloud
'''
If working in a multi-zone region like eu-de, the Zone must also be passed in via IBMCLOUD_ZONE.
'''

Reply:
@clnperez you are right.  Multi-zone communication is possible by setting up a GRE tunnel between the two DCs. This is a manual setup performed by the user. 
We'll need to investigate all the nuances with multi-zone kind of a setup in PowerVS and imho better to target it as future roadmap item.

Reply:
i think this is fine as long as we document it.